machine/v1beta1/types_vsphereprovider.go

```diff
@@ -70,6 +70,13 @@ type VSphereMachineProviderSpec struct {
 	// When using LinkedClone, if no snapshots exist for the source template, falls back to FullClone.
 	// +optional
 	CloneMode CloneMode `json:"cloneMode,omitempty"`
+	// Disks is a list of non OS disks to be attached to the VM.
+	// +openshift:enable:FeatureGate=VSphereMultiDisk
+	// +openshift:validation:FeatureGateAwareMaxItems:featureGate="",maxItems=0
+	// +openshift:validation:FeatureGateAwareMaxItems:featureGate=VSphereMultiDisk,maxItems=15
```

Comment:
is a max of 15 related to any documented limitation?

Reply:
just put it in place for now since scsi controller can only house up to 16 if I remember correctly (so 1 for OS disk).  Current design was to keep it simple and we just auto add to primarly scsi controller.

Reply:
This limit will need to be explained in the godoc

Reply:
I will update godoc to contain this info.